src/components/DropdownMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Person } from "../types/Person";
+
+type Props = {
+  filteredPeople: Person[];
+  onSelected: (person: Person) => void;
+}
+
+export const DropdownMenu: React.FC<Props> = React.memo(({ filteredPeople, onSelected }) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {filteredPeople.length > 0 ? (
+          filteredPeople.map((person: Person) => (
+            <div
+              className="dropdown-item"
+              data-cy="suggestion-item"
+              key={person.name}
```

Comment:
Using `person.name` as a key might not be unique if there are people with the same name. Consider using a unique identifier like `person.id` if available.